library/core/src/prelude/v1.rs

```diff
@@ -53,7 +53,6 @@ pub use crate::hash::macros::Hash;
 
 #[stable(feature = "builtin_macro_prelude", since = "1.38.0")]
 #[allow(deprecated)]
-#[doc(no_inline)]
```

Comment:
Was this removed intentionally?
All of the macros it applies to have public original location, so the prelude documentation *can* show them as reexports,  if there's a desire to do that (which I think there is, because all non-macro items are still marked with `no_inline`).

Reply:
So, I was having this weird error: https://github.com/rust-lang/rust/pull/92456#issuecomment-1003435516. I can try `no_inline`-ing again, but I suspect it will do the same? I'll admit I don't understand why it fails: could that be some `bootstrap`-related error?